tests/conftest.py

```diff
@@ -36,6 +37,33 @@
 from saleor.site.models import AuthorizationKey, SiteSettings
 
 
+def pytest_collection_modifyitems(items):
```

Comment:
I'd love to merge this without this fixture. I fear this will immediately become unmaintained and will result in an unpleasant surprise in future.

Reply:
This fixture ignores deprecated warning in django-measurement and django-impersonate. Without this, there still will be amount of warnings in CI.

Or just disable all deprecation warning by default? But this may bypass some useful warning as well.

Reply:
I'd rather see more warnings than necessary than accidentally miss some of the ones I care about. Does `pytest -Wd` still display the ones ignored here?

Reply:
No, it looks python's -Wd option won't override the markers added in ```pytest_collection_modifyitems```.